.github/workflows/azure-site-extension.yml

```
@@ -0,0 +1,78 @@
+name: Azure Site Extension
+
+on:
+  workflow_dispatch:
+
+env:
+  SPEC_FILE_TEMPLATE: 'NewRelic.Azure.WebSites.Extension.NodeAgent.nuspec'
+
+jobs:
+  setup_tools:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup dotnet '6.0.x'
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '6.0.x'
+      - name: Display dotnet version
+        run: dotnet --version
+      - name: Display NuGet version
+        run: nuget ?
+
+  create_extension_bundle:
+    runs-on: windows-latest
+    needs:
+      - setup_tools
+
+    strategy:
+      matrix:
+        node-version: ['18.x', '20.x'] # Node 22 isn't yet available for Windows apps in Azure
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Find agent version
+        run: |
+          $env:npm_agent_version = npm view newrelic version 
+          echo "AGENT_VERSION=$env:npm_agent_version" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Set package filename
+        run: |
+          echo "PACKAGE_FILENAME=NewRelic.Azure.WebSites.Extension.NodeAgent.${{env.AGENT_VERSION}}-${{ matrix.node-version }}" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Verify environment vars # because we can't access GH env vars until the next step
+        run: |
+          echo "Agent version: ${{ env.AGENT_VERSION }}"
+          echo "Package filename: ${{ env.PACKAGE_FILENAME }}"
+
+      - name: Install agent
+        working-directory: cloud-tooling/azure-site-extension/Content
+        run: |
+          npm i --prefix Content newrelic@${{ env.AGENT_VERSION }}
```

```suggestion
          npm i --prefix . newrelic@${{ env.AGENT_VERSION }}
```

This ends up creating a subfolder in Content called content